Tests/NetworkingTests/Mocks/MockAuthenticationProvider.swift

```diff
@@ -1,6 +1,11 @@
 import Foundation
 @testable import Networking
 
-final class MockAuthenticationProvider: AuthenticationProvider {
-  var bearerToken: String = "testBearerToken"
+final class MockAuthorizationProvider: AuthorizationProvider {
+    private(set) var didFetchToken: String?
```

Comment:
Is the name for this correct according to what we use it for in tests?

Reply:
`serviceTokenScope` or just `tokenScope`?

Reply:
How about `didFetchTokenWithScope`?

Reply:
the `did` implies a Bool test to me but I'm not going to hold this up for that, to your discretion 🙂

Reply:
`requestedTokenScope`?

Reply:
perhaps the latter to keep naming consistent 🙂

Reply:
Done ✅